tests/README.md

```diff
@@ -153,30 +163,81 @@ In order to run the tests in GCP, you need a few things:
   - this assumes that your GKE cluster is using private nodes. If using public nodes, you don't need this.
 - GCP VM and firewall rule to send ingress traffic to GKE
 
+To just set up the VM with no router (this will not run the tests):
+
+```makefile
+make create-and-setup-vm
+```
+
+Otherwise, you can set up the VM, router, and run the tests with a single command. See the options in the sections below.
+
+By default, the tests run using the version of NGF that was `git cloned` during the setup. If you want to make
+incremental changes and copy your local changes to the VM to test, you can run
+
+```makefile
+make sync-files-to-vm
+```
+
+#### Functional Tests
+
 To set up the GCP environment with the router and VM and then run the tests, run the following command:
 
 ```makefile
 make setup-gcp-and-run-tests
 ```
 
-If you just need a VM and no router (this will not run the tests):
+To use an existing VM to run the tests, run the following
 
 ```makefile
-make create-and-setup-vm
+make run-tests-on-vm
+```
+
+#### NFR tests
+
+To set up the GCP environment with the router and VM and then run the tests, run the following command:
+
+
+```makefile
+make setup-gcp-and-run-nfr-tests
 ```
 
 To use an existing VM to run the tests, run the following
 
 ```makefile
-make run-tests-on-vm
+make nfr-test
 ```
 
+##### Longevity testing
+
+This test is run on its own (and also not in a pipeline) due to its long-running nature. It will run for 4 days before
+the tester must collect the results and complete the test.
+
+To start the longevity test, set up your VM (`create-and-setup-vm`) and run
+
+```makefile
+make start-longevity-test
+```
+
+> Note: If you want to re-run the longevity test, you need to clear out the `cafe.example.com` entry from the `/etc/hosts` file on your VM.
+
+You can verify the test is working by checking nginx logs to see traffic flow, and check that the cronjob is running and redeploying apps.
+
+To complete the longevity test and collect results, first visit the [GCP Monitoring Dashboards](https://console.cloud.google.com/monitoring/dashboards) page and select the `NGF Longevity Test` dashboard. Take PNG screenshots of each chart for the time period in which your test ran, and save those to be added to the results file.
+
+Next, run:
```

Comment:
how do we know when to start collecting results? will those wrk processes be gracefully shutdown?

Reply:
They run for 4 days. I can add that timeframe in here.

Reply:
does it mean the person who runs in need to setup a timer for 4 days and than start collecting the results after?

could we make it obvious for them when start collecting the results?

For example, those wrks scripts should exit in 4 days, right? could we find a way to see that those scripts exited ? or warn users when they try to collect, but those wrk scripts are still running

Reply:
User can ssh onto the VM and run `ps aux | grep wrk`. I can add that note